technische-universitat-dresden-finanzwirtschaft-und-finanzdienstleistungen-note.csl

```diff
@@ -0,0 +1,243 @@
+        <text macro="title-short"/>
+        <date variable="issued" prefix=" ">
+          <date-part name="year"/>
+        </date>
+      </group>
+      <choose>
+        <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+          <group delimiter=", " prefix=" " suffix=", ">
+            <text macro="title"/>
+            <text macro="edition"/>
+            <text macro="editor"/>
+          </group>
+          <text macro="publisher" prefix=" "/>
+        </if>
+        <else-if type="chapter paper-conference" match="any">
+          <text macro="title" prefix=" " suffix=","/>
+          <group prefix=" " delimiter=" ">
+            <text term="in" text-case="lowercase" suffix=":"/>
+            <text macro="editor"/>
+            <text variable="container-title" font-style="normal" suffix=","/>
+            <text variable="collection-title" suffix="."/>
+            <text variable="event" suffix="."/>
+            <group delimiter=", ">
+              <text macro="publisher" prefix=" "/>
+              <text macro="pages"/>
+            </group>
+          </group>
+        </else-if>
+        <else-if type="thesis">
+          <group delimiter=", " prefix=" ">
+            <text macro="title"/>
+            <text variable="genre"/>
+            <text macro="publisher"/>
+          </group>
+        </else-if>
+        <else>
+          <group suffix=",">
+            <text macro="title" prefix=" "/>
+            <text macro="editor" prefix=" "/>
+          </group>
+          <group prefix=" in: ">
+            <choose>
+              <if type="interview" match="any">
+                <text variable="medium" suffix=", "/>
+              </if>
+              <else-if type="entry-dictionary webpage post-weblog" match="any">
+                <text variable="container-title"/>
+              </else-if>
+              <else>
+                <text variable="container-title" suffix=", "/>
+              </else>
+            </choose>
+            <group delimiter=", ">
+              <group delimiter=" ">
+                <text term="volume" form="short"/>
+                <text variable="volume" suffix=","/>
+              </group>
+              <group delimiter=" ">
+                <text term="issue" form="short"/>
+                <text variable="issue"/>
+              </group>
+            </group>
+            <group>
+              <choose>
+                <if type="article-newspaper interview" match="any">
+                  <date form="numeric" variable="issued"/>
+                </if>
+              </choose>
+            </group>
+            <group prefix=", ">
+              <label suffix=" " variable="page" form="short"/>
+              <text variable="page"/>
+            </group>
+          </group>
+        </else>
+      </choose>
+      <text macro="access" suffix="."/>
+    </layout>
+  </bibliography>
+  <locale xml:lang="de">
```

Comment:
while this is perfectly valid, could you move this section (i.e. the locale) right behind the info section, i.e. after line 23? That's where it's in all other styles.